src/components/textbox/textbox.spec.js

```diff
@@ -36,6 +38,37 @@ describe("Textbox", () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it("renders a counter", () => {
+    const wrapper = mount(<Textbox value="test string" characterLimit="100" />);
+
+    expect(wrapper.find(CharacterCount).text()).toBe("11/100");
+  });
+
+  it("renders a counter with an overlimit warning", () => {
+    const wrapper = mount(
+      <Textbox
+        value="test string"
+        characterLimit="10"
+        enforceCharacterLimit={false}
+        warnOverLimit
+      />
+    );
+
+    assertStyleMatch(
+      {
+        color: "#C7384F",
```

Comment:
Is this value comes from our theme object? If so it would be good to use baseTheme here as well